packages/cysync-core/src/dialogs/Send/context/index.tsx

```diff
@@ -662,19 +663,48 @@ export const SendDialogProvider: FC<SendDialogContextProviderProps> = ({
       return lang.strings.send.recipient.amount.error;
     }
 
-    const xrpValdation =
+    const xrpValidation =
       transaction?.validation as IPreparedXrpTransaction['validation'];
 
-    if (xrpValdation?.isBalanceBelowXrpReserve) {
-      return lang.strings.send.recipient.amount.balanceBelowXrpReserve;
+    if (xrpValidation.isBalanceBelowXrpReserve && selectedAccount) {
+      const reserveBalance = selectedAccount.spendableBalance
+        ? new BigNumber(selectedAccount.balance)
+            .minus(selectedAccount.spendableBalance)
+            .toString()
+        : '1000000'; // 1 XRP
+
+      const { amount: _amount, unit } = getParsedAmount({
+        coinId: selectedAccount.parentAssetId,
+        assetId: selectedAccount.assetId,
+        unitAbbr:
+          selectedAccount.unit ??
+          getDefaultUnit(selectedAccount.parentAssetId, selectedAccount.assetId)
+            .abbr,
+        amount: reserveBalance,
+      });
+
+      return `${lang.strings.send.recipient.amount.balanceBelowReserveBalance}${_amount} ${unit.abbr}.`;
```

Comment:
Instead of appending values at the end, use proper format strings and then use `ParseLangString` function